p/api/fever.php

```diff
@@ -0,0 +1,894 @@
+<?php
+/**
+ * Fever API for FreshRSS
+ * Version 0.1
+ * Author: Kevin Papst / https://github.com/kevinpapst
+ *
+ * Inspired by:
+ * TinyTinyRSS Fever API plugin @dasmurphy
+ * See https://github.com/dasmurphy/tinytinyrss-fever-plugin
+ */
+
+// refresh is not allowed yet, probably we find a way to support it later
+if (isset($_REQUEST["refresh"])) {
+	exit;
+}
+
+// ================================================================================================
+// BOOTSTRAP FreshRSS
+require(__DIR__ . '/../../constants.php');
+require(LIB_PATH . '/lib_rss.php');    //Includes class autoloader
+Minz_Configuration::register('system', DATA_PATH . '/config.php', FRESHRSS_PATH . '/config.default.php');
+
+// check if API is enabled globally
+FreshRSS_Context::$system_conf = Minz_Configuration::get('system');
+if (!FreshRSS_Context::$system_conf->api_enabled) {
+	Minz_Log::warning('serviceUnavailable() ' . debugInfo(), API_LOG);
+	header('HTTP/1.1 503 Service Unavailable');
+	header('Content-Type: text/plain; charset=UTF-8');
+	die('Service Unavailable!');
+}
+
+Minz_Session::init('FreshRSS');
+// ================================================================================================
+
+// this allows to overwrite the FeverAPI for special clients
+if (!function_exists('createFeverApiInstance')) {
+	function createFeverApiInstance() {
+		return new FeverAPI();
+	}
+}
+
+/**
+ * Class FeverAPI_FeedDAO for more feed functions than FreshRSS offers.
+ */
+class FeverAPI_FeedDAO extends FreshRSS_FeedDAO
+{
+	/**
+	 * @return FreshRSS_Feed
+	 */
+	public function getLastUpdatedFeed() {
```

Comment:
Seems to be very similar to `FreshRSS_FeedDAO::listFeedsOrderUpdate()`, to which we could add a `limit` parameter if needed.

Reply:
I checked that method before, but I didn't support limit. If you add that I can switch. Can you push to this PR?

Reply:
Not tested, but https://github.com/FreshRSS/FreshRSS/pull/1836/commits/2c271f0b725dd978b28bf86f964ee7a396e6278b should do the job :-)

Reply:
Works, thanks! I don't really get the calculation happening there as I had to use `listFeedsOrderUpdate(-1, 1)` - but I guess thats not a problem?